### PR TITLE
Dockerize app

### PR DESCRIPTION
**Pull request by int128**
This pull request adds `Dockerfile` and example `docker-compose.yml`. It allows users can quickly run jira-matter-bridge by Docker.

README assumes that the Docker image is available as `vrenjith/jira-matter-bridge` in Docker Hub. After merged, please login to [Docker Hub](https://hub.docker.com) and setup the automated build from your repository in order to publish the Docker image. Let me know if you have any issue on setting up.

Thank you for the great app.

### CODE REVIEW

**Comment by zephyrweb** [comment]
You can also setup a Dockerfile like this:
```
FROM node:argon
# Create app directory
RUN mkdir -p /usr/src/app
WORKDIR /usr/src/app
# Install app dependencies
COPY package.json /usr/src/app/
RUN npm install
# Bundle app source
COPY . /usr/src/app

EXPOSE 3000
CMD [ "npm", "start" ]
```
And link it in docker-compose.yml like this:
```
jira:
  build: jira
  ports:
    - "3000:3000"
  links:
    - web:web
  environment:
    - MATTERMOST_SERVER_PORT=80
    - MATTERMOST_SERVER=web
```

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+FROM node:6-onbuild
+EXPOSE 3000
```
Comment (zephyrweb) [inline Dockerfile]: Needs a CMD or an ENTRYPOINT, right?

Comment (vrenjith) [inline Dockerfile]: @int128  
Can you check if the comments from @zephyrweb can be incorporated so that I can merge it?

**Comment by vrenjith** [comment]
I merged it.

**Comment by int128** [comment]
Would you create an automated build on [Docker Hub](https://hub.docker.com/)?

**Pull request merged**